types/culori/culori-tests.ts

```diff
@@ -0,0 +1,390 @@
+culoriRequire.useMode(culoriRequire.modeA98); // $ExpectType ConvertFn<"a98">
+culoriRequire.useMode(culoriRequire.modeCubehelix); // $ExpectType ConvertFn<"cubehelix">
+culoriRequire.useMode(culoriRequire.modeDlab); // $ExpectType ConvertFn<"dlab">
+culoriRequire.useMode(culoriRequire.modeDlch); // $ExpectType ConvertFn<"dlch">
+culoriRequire.useMode(culoriRequire.modeHsi); // $ExpectType ConvertFn<"hsi">
+culoriRequire.useMode(culoriRequire.modeHsl); // $ExpectType ConvertFn<"hsl">
+culoriRequire.useMode(culoriRequire.modeHsv); // $ExpectType ConvertFn<"hsv">
+culoriRequire.useMode(culoriRequire.modeHwb); // $ExpectType ConvertFn<"hwb">
+culoriRequire.useMode(culoriRequire.modeItp); // $ExpectType ConvertFn<"itp">
+culoriRequire.useMode(culoriRequire.modeJab); // $ExpectType ConvertFn<"jab">
+culoriRequire.useMode(culoriRequire.modeJch); // $ExpectType ConvertFn<"jch">
+culoriRequire.useMode(culoriRequire.modeLab); // $ExpectType ConvertFn<"lab">
+culoriRequire.useMode(culoriRequire.modeLab65); // $ExpectType ConvertFn<"lab65">
+culoriRequire.useMode(culoriRequire.modeLch); // $ExpectType ConvertFn<"lch">
+culoriRequire.useMode(culoriRequire.modeLch65); // $ExpectType ConvertFn<"lch65">
+culoriRequire.useMode(culoriRequire.modeLchuv); // $ExpectType ConvertFn<"lchuv">
+culoriRequire.useMode(culoriRequire.modeLrgb); // $ExpectType ConvertFn<"lrgb">
+culoriRequire.useMode(culoriRequire.modeLuv); // $ExpectType ConvertFn<"luv">
+culoriRequire.useMode(culoriRequire.modeOkhsl); // $ExpectType ConvertFn<"okhsl">
+culoriRequire.useMode(culoriRequire.modeOkhsv); // $ExpectType ConvertFn<"okhsv">
+culoriRequire.useMode(culoriRequire.modeOklab); // $ExpectType ConvertFn<"oklab">
+culoriRequire.useMode(culoriRequire.modeOklch); // $ExpectType ConvertFn<"oklch">
+culoriRequire.useMode(culoriRequire.modeP3); // $ExpectType ConvertFn<"p3">
+culoriRequire.useMode(culoriRequire.modeProphoto); // $ExpectType ConvertFn<"prophoto">
+culoriRequire.useMode(culoriRequire.modeRec2020); // $ExpectType ConvertFn<"rec2020">
+culoriRequire.useMode(culoriRequire.modeRgb); // $ExpectType ConvertFn<"rgb">
+culoriRequire.useMode(culoriRequire.modeXyb); // $ExpectType ConvertFn<"xyb">
+culoriRequire.useMode(culoriRequire.modeXyz50); // $ExpectType ConvertFn<"xyz50">
+culoriRequire.useMode(culoriRequire.modeXyz65); // $ExpectType ConvertFn<"xyz65">
+culoriRequire.useMode(culoriRequire.modeYiq); // $ExpectType ConvertFn<"yiq">
+culoriRequire.useParser((str) => ({ mode: "rgb", r: 0, g: 0, b: 0 })); // $ExpectType undefined
+culoriRequire.useParser("--oklab", "oklab"); // $ExpectType undefined
+culoriRequire.wcagContrast("#0000ff", "#008080"); // $ExpectType number
+culoriRequire.wcagLuminance("#ffffff"); // $ExpectType number
+culoriRequire.xyb("#ffffff"); // $ExpectType Xyb | undefined
+culoriRequire.xyz50("#ffffff"); // $ExpectType Xyz50 | undefined
+culoriRequire.xyz65("#ffffff"); // $ExpectType Xyz65 | undefined
+culoriRequire.yiq("#ffffff"); // $ExpectType Yiq | undefined
```

Comment:
This list came from just copying + pasting from the actual `bundled/culori.cjs` file in the package, and plugging in simple values. The type assertions are all very basic, and the main goal of these tests were to just test that everything was exported in the CJS bundle correctly that exists in the ESM version.

As a bonus, because this file is 100% generated from the ESM version, they’re also testing the ESM version too! So the tests in general got a lot more cases